src/WatchKit/WKDefs.cs

```diff
@@ -13,6 +13,7 @@
 
 namespace WatchKit {
 	[iOS (8,2)]
+	[Unavailable (PlatformName.iOS)]
```

Comment:
not sure how the IDE will interpret (and show) this mix of `[Introduced]` and `[Available]` ?!?

Reply:
The [Unavailable] makes the generator not generate anything (for iOS, still generates for watchOS), so there's no mix in the final source code.

The source code used for iOS is the manual code elsewhere in this PR.